src/modules/class/application/create-class.dto.ts

```diff
@@ -0,0 +1,35 @@
+
+import { ApiProperty } from "@nestjs/swagger";
+import { IsArray, IsNotEmpty, IsNumber, IsOptional, IsString } from "class-validator"
```

Comment:
```suggestion
import { IsArray,
         IsNotEmpty,
         IsNumber,
         IsOptional,
         IsString
        } from "class-validator"
```
Com as alterações sugeridas abaixo, será preciso importar esses decorators.